data/saas/saas_connector_registry.toml

```diff
@@ -159,3 +159,9 @@ config = "data/saas/config/recharge_config.yml"
 dataset = "data/saas/dataset/recharge_dataset.yml"
 icon = "data/saas/icon/default.svg"
 human_readable = "Recharge"
+
+[universal_analytics]
+config = "data/saas/config/universal_analytics_config.yml"
+dataset = "data/saas/dataset/universal_analytics_dataset.yml"
+icon = "data/saas/icon/google_analytics.svg"
```

Comment:
![Screen Shot 2023-02-01 at 3 48 07 PM](https://user-images.githubusercontent.com/9755598/216171159-bdcdba91-1169-46c2-a477-5f0c6d9e2596.png)

Is this an OK logo to use @saiatethyca? I didn't see a logo without the words "Google Analytics", so I just pulled the three bars off the logo with the name in it, and adjusted the size to be 32x32.

Reply:
Yeah, this makes sense. I don't see a different logo for universal analytics vs google analytics. This is perfect, Thanks!

Reply:
great!